roles/cdh_common/molecule/default/verify.yml

```diff
@@ -0,0 +1,27 @@
+---
+# This is an example playbook to execute Ansible tests.
+
+- name: Verify
+  hosts: all
+  gather_facts: false
+  vars_files:
+    - ../../defaults/main.yml
```

Comment:
Is there another option besides this relative path? Would prefer it to be explicit and more readable